.ci/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-version=$(awk '/^ENV RAPIDPRO_VERSION/ { print $3 }' rapidpro-docker/Dockerfile)
+version=$(curl -s https://api.github.com/repos/praekeltfoundation/rapidpro/tags  | jq -r '.[0].name')
```

Comment:
Oh, I see it gets us the latest tag. Not really the same thing, but maybe it will be sufficient?